src/types/IUser.ts

```diff
@@ -2,9 +2,13 @@ export interface IUser {
   id: number;
   name: string;
   email: string;
+  password: string;
 }
 
 export interface ICreateUser {
```

Comment:
ICreateUserDto

Reply:
i will skip it for while, i need time for it